src/neo/SmartContract/Native/Tokens/NepAccountState.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Neo.SmartContract.Native.Tokens
 {
-    public class Nep5AccountState : IInteroperable
+    public class NepAccountState : IInteroperable
```

Comment:
Maybe `NepAccountState` will be quite generic.
Perhaps it could be something like `BalanceAccountState` or something related to BigIntenger `BigIntengerAccountState`

Reply:
It is just an idea, fell free to keep as it as well because maybe it is good like this.

Reply:
@erikzhang just commit the pure name `AccountState` on https://github.com/neo-project/neo/commit/ce0234d7f9775eb26abb147283adb62795d933c3